examples/consul-image/README.md

```diff
@@ -63,17 +63,13 @@ Your code should look more like this:
   "provisioners": [{
     "type": "shell",
     "inline": [
-      "git clone --branch <MODULE_VERSION> https://github.com/gruntwork-io/consul-gcp-module.git /tmp/consul-gcp-module",
-      "/tmp/consul-gcp-module/modules/install-consul/install-consul --version {{user `consul_version`}}",
-      "/tmp/consul-gcp-module/modules/install-dnsmasq/install-dnsmasq"
+        "git clone --branch `<MODULE_VERSION>` https://github.com/hashicorp/terraform-google-consul.git  /tmp/terraform-google-consul",
+        "/tmp/terraform-google-consul/modules/install-consul/install-consul --version {{user `consul_version`}}",
+        "/tmp/terraform-google-consul/modules/install-dnsmasq/install-dnsmasq"
     ],
     "pause_before": "30s"
   }]
 }
 ```
 
-You should replace `<MODULE_VERSION>` in the code above with the version of this Module that you want to use (see
-the [Releases Page](https://github.com/hashicorp/terraform-google-consul/tree/master/releases) for all available versions). That's because for production usage, you should always
-use a fixed, known version of this Module, downloaded from the official Git repo. On the other hand, when you're 
-just experimenting with the Module, it's OK to use a local checkout of the Module, uploaded from your own 
-computer.
\ No newline at end of file
+Get the version tag from the Consul [releases page](https://releases.hashicorp.com/consul/). You should replace `<MODULE_VERSION>` in the code above with the version tag of the module that you want to use. For production usage, you should always use a fixed, known version of this module, downloaded from the official Git repo. If you're just experimenting with the module, it's OK to use a local checkout of the module, uploaded from your own computer.
```

Comment:
The releases page refers to this module's releases, not to Consul's releases, so it should link to https://github.com/hashicorp/terraform-google-consul/releases.

Reply:
D'oh, of course, sorry about that @josh-padnick !
Would you like me to use the URL pointing directly to the list of tags:
https://github.com/hashicorp/terraform-google-consul/tags

Reply:
No worries. https://github.com/hashicorp/terraform-google-consul/releases is better because it shows release notes inline.